templates/new_service/.github/workflows/delete-review-app.yml

```diff
@@ -0,0 +1,105 @@
+name: Delete Review App
+
+on:
+  pull_request:
+    types: [closed]
+    branches: [main]
+  workflow_dispatch:
+    inputs:
+      pr_number:
+        description: PR number of review app to delete
+        required: true
+        type: string
+jobs:
+  delete-review-app:
+    name: Delete Review App ${{ github.event.pull_request.number }}
+    concurrency: deploy_review_${{ github.event.pull_request.number }}
+    if: contains(github.event.pull_request.labels.*.name, 'deploy-aks') ||  ${{ github.event_name }} == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    environment: review
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: set PR_NUMBER
+        id: config
+        run: |
+          if [ ${{ github.event_name }} == 'workflow_dispatch' ]; then
+            PR_NUMBER=${{ github.event.inputs.pr_number }}
+          else
+            PR_NUMBER=${{ github.event.pull_request.number }}
+          fi
+          echo "PR_NUMBER=$PR_NUMBER" >> $GITHUB_ENV
+
+      - uses: hashicorp/setup-terraform@v3
+        with:
+          terraform_version: 1.6.4
+          terraform_wrapper: false
+
+      - uses: DFE-Digital/github-actions/set-kubelogin-environment@master
+        with:
+          azure-credentials: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: Terraform Destroy
+        run: |
+          make ci review terraform-destroy PR_NUMBER=${{ env.PR_NUMBER }}
+        env:
+          PR_NUMBER: ${{ env.PR_NUMBER }}
+
+      - name: Post Pull Request Comment
+        if: ${{ github.event_name == 'pull_request' }}
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:
+          header: aks
+          message: |
+            Review app refer serious misconduct deployed to <https://#SERVICE_NAME#-${{ env.PR_NUMBER }}.test.teacherservices.cloud> was deleted
+
+      - uses: Azure/login@v2
+        with:
+          creds: ${{ secrets.azure_credentials }}
+
```

Comment:
we will need gcp login. e.g. https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/main/.github/workflows/delete-review-app.yml#L25 but using repo variables for project_id and wip i.e. vars.project_id and vars.gcp_wip. Only run if both vars aren't empty

Reply:
why can  i find the vars.project_id and vars.gcp_wip variables ?

Reply:
we would add them to the repo if it uses gcp e.g. https://github.com/DFE-Digital/apply-for-qualified-teacher-status/settings/variables/actions

Reply:
added gcp login